### PR TITLE
Codechange: [AzurePipeline] add developer files section in manifest

### PR DESCRIPTION
**Pull request by glx22**
For now it just adds debug files. As said in https://github.com/OpenTTD/website/pull/111#issuecomment-550005376, we need to decide which files should be moved to `DEV_FILES`.

### CODE REVIEW

File: azure-pipelines/manifest.sh [hunk 1]
```diff
@@ -31,29 +31,44 @@ DATE=$(cat .release_date | tr ' ' T | sed 's/TUTC/:00-00:00/')
...
-    if [ ! -e ${FOLDER}/$i.md5sum ] || [ ! -e ${FOLDER}/$i.sha1sum ] || [ ! -e ${FOLDER}/$i.sha256sum ]; then
-        echo "ERROR: missing checksum file for ${i}" 1>&2
-        error="y"
-        continue
-    fi
+FILES=files
+DEV_FILES=dev_files
+for i in $(ls ${FOLDER} | grep -v ".txt$\|.md$\|sum$" | sort); do
+    if [ -n "$(echo $i | grep pdb.xz)" ]; then DEV_FILES+=" $i"; continue; fi
+    if [ -n "$(echo $i | grep dbg.deb)" ]; then DEV_FILES+=" $i"; continue; fi
```
Comment (LordAro) [inline azure-pipelines/manifest.sh]: I think a case statement would be better here

File: azure-pipelines/manifest.sh [hunk 2]
```diff
@@ -31,29 +31,44 @@ DATE=$(cat .release_date | tr ' ' T | sed 's/TUTC/:00-00:00/')
...
-    if [ -n "$(echo $i | grep pdb.xz)" ]; then continue; fi
-    if [ -n "$(echo $i | grep dbg.deb)" ]; then continue; fi
 
-    if [ ! -e ${FOLDER}/$i.md5sum ] || [ ! -e ${FOLDER}/$i.sha1sum ] || [ ! -e ${FOLDER}/$i.sha256sum ]; then
-        echo "ERROR: missing checksum file for ${i}" 1>&2
-        error="y"
-        continue
-    fi
+FILES=files
+DEV_FILES=dev_files
```
Comment (LordAro) [inline azure-pipelines/manifest.sh]: this makes the argument parsing a bit weird. Maybe just pass `files` & `dev_files` to `output_files` directly?

```bash
output_files dev_files $DEV_FILES
```

**Review by LordAro — DISMISSED** [review]

**Comment by TrueBrain** [comment]
Please use a better commit message :) This is not descriptive what the change is (only why you want it :P)

File: azure-pipelines/manifest.sh [hunk 3]
```diff
@@ -31,29 +31,54 @@ DATE=$(cat .release_date | tr ' ' T | sed 's/TUTC/:00-00:00/')
...
-    echo "  md5sum: $(cat ${FOLDER}/$i.md5sum | cut -d\  -f1)" >> manifest.yaml
-    echo "  sha1sum: $(cat ${FOLDER}/$i.sha1sum | cut -d\  -f1)" >> manifest.yaml
-    echo "  sha256sum: $(cat ${FOLDER}/$i.sha256sum | cut -d\  -f1)" >> manifest.yaml
+FILES=
+DEV_FILES=
+for filename in $(ls ${FOLDER} | grep -v ".txt$\|.md$\|sum$" | sort); do
+    case ${filename} in
+        *docs*   |\
+        *dbg.deb |\
+        *pdb.xz  )
```
Comment (LordAro) [inline azure-pipelines/manifest.sh:47]: needs sources adding to it, other than that, is all good

**Review by LordAro — APPROVED** [review]

**Pull request merged**